.github/workflows/backup_audit_logs.yml

```
@@ -17,11 +17,19 @@ jobs:
       - name: Install dependencies
         run: npm i
 
-      - name: Fetch audit logs
-        run: node bot.mjs
+      - name: Linter
+        run: npm run lint
+
+      - name: Prettier
+        run: npm run format
```

```suggestion
      - name: Lint Check
      	if: always()
        run: npm run lint-check

      - name: Type Check
      	if: always()
        run: npm run type-check
```
Always run both checks just to give the user more info. These also need to be checks, not just "lint" and "format", as that won't report any errors (as they won't fail).